openml/tasks/task.py

```diff
@@ -29,11 +29,6 @@ def __init__(self, task_id, task_type, data_set_id, target_feature,
         if cost_matrix is not None:
             raise NotImplementedError("Costmatrix")
 
-    def __str__(self):
```

Comment:
I removed this to had all objects consistent. There is an issue about adding __repr__ to all objects to fix this.